privacy_analytics/templates/privacy_analytics/dashboard.html

```diff
@@ -1,113 +1,149 @@
+</section>
+
+<section class="section">
+    <div class="level">
+        <div class="level-item level-left">
+            <form method="GET">
+                <input type="hidden" name="before" value="{{ start }}"></input>
+                <input type="hidden" name="action" value="clear">
+                <button class="button is-danger is-outlined">Clear Stored Analytics</button>
+            </form>
         </div>
     </div>
-</div>
-<br><br>
-<div class="columns">
-    <div class="column is-8">
-        <table class="table is-fullwidth">
-            <thead>
-                <tr>
-                    <th>Path</th>
-                    <th>Count</th>
-                    <th>Percent</th>
-                    <th></th>
-                </tr>
-            </thead>
-            <tbody>
-                {% for page in pages %}
-                <tr>
-                    <td class="is-narrow">
-                        <form method="GET">
-                            <input type="hidden" value="{{page.path}}" name="path">
-                            <input type="hidden" value="{{page.from}}" name="from">
-                            <input type="hidden" value="{{page.until}}" name="until"><a
-                                href="{{page.path}}">{{page.path|truncatechars:30}}</a>
-                            <button type="submit" class="button is-small is-text">&rarr;</button></form>
-                    </td>
-                    <td class="is-narrow">{{page.n}}</td>
-                    <td class="is-narrow">{{page.percent|intcomma|floatformat:"-1"}}%</td>
-                    <td><progress class="progress is-link is-large" value="{{page.percent}}"
-                            max="100">{{page.percent}}%</progress></td>
-                </tr>
-                {% endfor %}
-            </tbody>
-        </table>
-    </div>
-    <div class="column is-4">
-        <table class="table is-fullwidth">
-            <thead>
-                <tr>
-                    <th>Referrer</th>
-                    <th>Count</th>
-                </tr>
-            </thead>
-            <tbody>
-                {% for referrer in referrers %}
-                <tr>
-                    <td>
-                        {% if referrer.referrer %}
-                        <a href="{{referrer.referrer}}">{{referrer.referrer|truncatechars:30}}</a>
-                        {% else %}
-                        No Referrer
-                        {% endif %}
-                    </td>
-                    <td>{{referrer.n}}</td>
-                </tr>
-                {% endfor %}
-            </tbody>
-        </table>
-    </div>
-</div>
-<hr>
-<form method="POST">
-    {% csrf_token %}
-    <input type="hidden" name="before" value="{{from}}">
-    <input type="hidden" name="action" value="clear">
-    <button type="submit" class="button is-danger is-outlined">Clear Data Before {{from}}</button>
-</form>
+</section>
 {% endblock %}
```

Comment:
This looks excellent, thanks for building all this. I haven't taken a look at the new interface in the browser, but I'll do so as soon as we move forward with the backend changes!

Reply:
It's pretty much a carbon-copy but I just wanted to neaten things up. I am thinking about doing a more intense dashboard mock-up. I will definitely get your thoughts once I whip something up!